ocfweb/docs/docs/staff/procedures/vhost.md

```diff
@@ -36,3 +36,27 @@ Then, whether or not the group has web virtual hosting, request the following DN
     hostname.Berkeley.EDU. IN MX 5 sandstorm.OCF.Berkeley.EDU.
 
 Use the domain requested by the group in place of `hostname`. We have a [reusable email template](http://templates.ocf.berkeley.edu/#hostmaster-add-mail) for making DNS mail requests.
+
+### Application hosting
+The group website should be reasonably developed (can be offsite during review only for this request) before approving it.
+
+You will need a `/admin` principle to modify apphosting entries.
+
+* Add the group account to the ocf LDAP group:
+
+      $ ldapvi cn=ocfdev
+      memberUid: ggroup
+      memberUid: GROUP_USERNAME
+
+* Add apphost entry: edit `~staff/vhost/vhost-app.conf`.
+* Wait for cronjob to update configurations (runs every 10 minutes).
+
+Once the cronjob completes, the application will be available at:
+
+    VHOST_NAME-berkeley-edu.apphost.ocf.berkeley.edu
```

Comment:
It might be good to specify that this variable for the subdomain comes from the file edited above, just to be clearer.